bluemix/helpers.js

```diff
@@ -10,218 +10,226 @@ var g = require('strong-globalize')();
+      // Generate docker file
+      this.enableDocker = this.options.docker;
+      // Generate toolchain files
+      this.enableToolchain = this.options.toolchain;
+      this.bluemix = this.enableManifest || this.enableDocker ||
+      this.enableToolchain || this.options.login || this.options.sso;
+      // Add default services
+      this.enableDefaultServices = !this.bluemix;
 
-  var done = this.async();
-  this.prompt([manifestPrompt]).then(function(answers) {
-    this.enableManifest = answers.enableManifest || this.enableManifest;
-    var prompts = [];
-    if (this.enableManifest) {
-      var manifestFilePath = path.resolve('.', 'manifest.yml');
-      var manifest = {};
-      if (fs.existsSync(manifestFilePath)) {
-        manifest = yaml.load(fs.readFileSync(manifestFilePath));
+      if (this.bluemix) {
+        this.log(g.f('Bluemix-specific configuration:'));
       }
 
-      var appMemoryPrompt = {
-        name: 'appMemory',
-        message: g.f('How much memory to allocate for the app?'),
-        default: manifest.memory || '256M',
-        filter: helpers.normalizeSize,
-        validate: helpers.validateAppMemory,
-        when: this.enableManifest,
-      };
-
-      var appInstancesPrompt = {
-        name: 'appInstances',
-        message: g.f('How many instances of app to run?'),
-        default: manifest.instances || 1,
-        validate: helpers.validateAppInstances,
-        when: this.enableManifest,
-      };
-
-      var appDomainPrompt = {
-        name: 'appDomain',
-        message: g.f('What is the domain name of the app?'),
-        default: manifest.domain || 'mybluemix.net',
-        validate: helpers.validateAppDomain,
-        when: this.enableManifest,
-      };
-
-      var appHostPrompt = {
-        name: 'appHost',
-        message: g.f('What is the subdomain of the app?'),
-        default: manifest.host || this.appName,
-        validate: helpers.validateAppHost,
-        when: this.enableManifest,
-      };
-
-      var appDiskQuotaPrompt = {
-        name: 'appDiskQuota',
-        message: g.f('How much disk space to allocate for the app?'),
-        default: manifest.disk_quota || '1G',
-        filter: helpers.normalizeSize,
-        validate: helpers.validateAppDiskQuota,
-        when: this.enableManifest,
+      var manifestPrompt = {
+        name: 'enableManifest',
+        message: g.f('Do you want to create manifest.yml?'),
+        type: 'confirm',
+        default: true,
+        // Only prompt if no explicit option such as --manifest or --docker
+        // is not present
+        when: !this.bluemix,
       };
 
-      prompts.push(appMemoryPrompt, appInstancesPrompt, appDomainPrompt,
-        appHostPrompt, appDiskQuotaPrompt);
-    }
-
-    var toolchainPrompt = {
-      name: 'enableToolchain',
-      message: g.f('Do you want to create toolchain files?'),
-      type: 'confirm',
-      default: true,
-      when: !this.bluemix,
+      var done = this.async();
```

Comment:
Not needed.

Reply:
@virkt25 It's get called on [Line 173](https://github.com/strongloop/generator-loopback/pull/363/files#diff-5ec15de4470e70e230e53c3a08375e18R173)